devel/git-crypt/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            AGWA git-crypt 0.6.0
+homepage                https://www.agwa.name/projects/git-crypt/
+categories              devel
+platforms               darwin
+maintainers             @nareshov
+license                 GPL-3
```

Comment:
According to https://github.com/AGWA/git-crypt/blob/0.6.0/git-crypt.cpp#L6,
this should be `{GPL-3+ OpenSSLException}`